interface/wx/treectrl.h

```diff
@@ -506,6 +506,12 @@ class wxTreeCtrl : public wxControl
     */
     virtual unsigned int GetIndent() const;
 
+    /**
+        Returns the current tree control spacing (the number of pixels between
+        the start and the Text).  Has no effect under wxMSW.
```

Comment:
Sorry, this doesn't seem quite correct to me:

1. Start of what? And why is the "Text" capitalized?
1. "Has no effect" doesn't make much sense for the accessor IMO (although it's OK for the setter).

Reply:
Okay, so I removed "has no effect" from the accessor and removed the capitalization of Text.  Other than that, I'm not entirely sure how to improve the description.  I basically copied it from the include/wx/treectrl.h header comments.  Personally, I don't actually understand what 'Spacing' means exactly in this particular case.  I only ran into this when porting something to wxPython 4.0 and these functions were missing.

Reply:
To be honest, I'm not too sure myself, but the treectrl sample has menu items to increase/decrease it, so it should be visible in it, I guess? From looking at the code it looks like it's the space before the actual text reserved for showing the button/checkbox before the item.

Reply:
Okay, I played around a bit with the sample to see what it does.  See what you think of my new description.